mainhandler/handlerequests.go

```diff
@@ -85,7 +87,15 @@ func (mainHandler *MainHandler) HandleWatchers(ctx context.Context) {
 		}
 	}()
 
-	watchHandler, err := watcher.NewWatchHandler(ctx)
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err)
+	}
+	ksStorageClient, err := kssc.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
question: can we safely assume that the operator is running inside a cluster?

Or do we have to use some `k8sinterface` function to load the config?

And if we can’t load a config, it reasonable to immediately abort here? Since without the config, running the Operator seems kinda pointless.

Reply:
1. We should not assume we are running in a cluster just for the purpose of debugging locally.
2. Yes, if the config is not found at all we should panic. In such a case we should call the `logger.L().Fatal()` function, not `panic`.